x-pack/platform/plugins/shared/ml/public/application/jobs/jobs_list/components/jobs_list/jobs_list.js

```diff
@@ -107,6 +109,12 @@ export class JobsListUI extends Component {
   }
 
   render() {
+    const { spaces, application } = this.props.kibana.services;
+    const shouldShowSpacesColumn =
+      spaces !== undefined &&
+      application.capabilities &&
+      application.capabilities.spaces?.manage === true &&
```

Comment:
I don't think this check is needed as it adds a very tight restriction.
If I create a new role with access to every kibana feature, it doesn't grant me this capability and so this column is missing.

This capability I believe is to control whether a user has permission to edit the spaces themselves, rather than being able to move saved objects about.

Reply:
Removed this check here [`cfeac4e` (#206696)](https://github.com/elastic/kibana/pull/206696/commits/cfeac4e786d192e1b487711662b2275f88f1d01c)